_resources/port1.0/group/xcodeversion-1.0.tcl

```diff
@@ -14,6 +14,11 @@
 options minimum_xcodeversions
 default minimum_xcodeversions {}
 
+# Xcode should be used for this port
+if {[info exists use_xcode]} {
+    use_xcode yes
```

Comment:
`default use_xcode yes` also works. Not sure which one is better since it's not standard between portgroups and I don't know the difference.

Reply:
`default` will only change the value if it wasn't set before. Since PortGroups are usually the first statements in Portfiles, it is unlikely that there already was a value set.

The only situation where a value could already have been set is if you included multiple PortGroups that set `use_xcode` – at the moment, we do not have such a situation, given that `use_xcode` defaults to no, and we only ever set it to yes.

In such situations, you could use `default` in one of the PortGroups and setting the option explicitly in the other to specify preference.

As it doesn't make a difference here, let's leave it as-is.